### PR TITLE
rhel setup: drop debuginfo-install

### PR DESCRIPTION
**Pull request by tomasmatus**
* [x] image-refresh centos-9-stream

fixes: #6606

### CODE REVIEW

File: images/scripts/rhel.setup [hunk 1]
```diff
@@ -329,9 +329,6 @@ fi
 # and more convenient interactive debugging
 dnf --setopt=install_weak_deps=False install -y cockpit
 
-# For debugging udisks/storaged crashes
-debuginfo-install -y udisks2
```
Comment (martinpitt) [inline images/scripts/rhel.setup:333]: Please only drop it on centos, it seems to work well on RHEL.

Comment (tomasmatus) [reply images/scripts/rhel.setup:333]: ack, "Let's kill that debuginfo-install." sounded too strinct to keep it on rhel 😄

**Comment by cockpituous** [comment]
image-refresh centos-9-stream done: https://github.com/cockpit-project/bots/commits/image-refresh-centos-9-stream-20240715-112442

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-centos-9-stream-01c3b1b8-20240715-110912/log.html

**Comment by martinpitt** [comment]
Looks good, thanks! Please trigger the image tests.

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**